assets/javascripts/discourse/connectors/flagged-post-below-controls/sift-flag-queue.js.es6

```diff
@@ -0,0 +1,38 @@
+import SiftMod from 'discourse/plugins/discourse-sift/admin/models/sift-mod';
+
+function genericError() {
+    bootbox.alert(I18n.t('generic_error'));
+}
+
+export default {
+
+    internalReportingDone: (reason) => {
```

Comment:
This is a bit odd. Using the fat arrow syntax means that `this` doesn't point to your singleton anymore. Additionally this doesn't seem to be used?

Reply:
That was the last attempt I made to get a common function that could be called, but could not get it to work.  Removed